spec/index.bs

```diff
@@ -2577,6 +2595,10 @@ interface GPUAdapter {
     ::
         Returns the value of {{GPUAdapter/[[adapter]]}}.{{adapter/[[fallback]]}}.
 
+    : <dfn>isXrCompatible</dfn>
+    ::
+        Returns the value of {{GPUAdapter/[[adapter]]}}.{{adapter/[[xrCompatible]]}}.
```

Comment:
I think we should just set this directly in requestAdapter rather than having a backing field for it, and same for `isFallbackAdapter`. (This is why we have `slot-backed attribute`, though I bet `isFallbackAdapter` was not the only case where we had something defined as a slot-backed attribute and then ignore that and have it return something else.)

Reply:
Yes, but also see [1] for the reason why this is actually going away entirely.